src/L2/L2AirdropV2.sol

```diff
@@ -0,0 +1,282 @@
+    }
+
+    /// @notice Set Merkle root for the airdrop process.
+    /// @param _merkleRoot Merkle root for the airdrop process.
+    /// @dev Only the owner can set the Merkle root.
+    function setMerkleRoot(bytes32 _merkleRoot) public onlyOwner {
+        require(_merkleRoot != 0, "L2AirdropV2: Merkle root can not be zero");
+        require(merkleRoot == 0, "L2AirdropV2: Merkle root already set");
+        merkleRoot = _merkleRoot;
+        airdropStartTime = block.timestamp;
+        emit MerkleRootSet(merkleRoot);
+    }
+
+    /// @notice Send the remaining LSK tokens to the Ecosystem Fund wallet.
+    /// @dev Only the owner can send the remaining LSK tokens to the Ecosystem Fund wallet.
+    function sendLSKToEcosystemWallet() public onlyOwner {
+        require(merkleRoot != 0, "L2AirdropV2: airdrop has not started yet");
+        require(
+            airdropStartTime + (HODLER_AIRDROPV2_DURATION * 1 days) < block.timestamp,
+            "L2AirdropV2: airdrop is not over yet"
+        );
+        uint256 balance = IL2LiskToken(l2LiskTokenAddress).balanceOf(address(this));
+        // reentrancy won't be an issue here because the L2 Lisk Token contract is trusted and managed by the team
+        // slither-disable-next-line reentrancy-no-eth
+        // slither-disable-next-line reentrancy-events
+        bool status = IL2LiskToken(l2LiskTokenAddress).transfer(ecosystemFundAddress, balance);
+        require(status, "L2AirdropV2: LSK token transfer to the Ecosystem Fund wallet failed");
+        emit LSKSentToEcosystemWallet(ecosystemFundAddress, balance);
+    }
+
+    /// @notice Check if the Lisk v4 address has claimed the airdrop for staking tier 1.
+    /// @param liskAddress The Lisk v4 address to check if it has claimed the airdrop for staking tier 1.
+    /// @return True if the Lisk v4 address has claimed the airdrop for staking tier 1, False otherwise.
+    function claimedStakingTier1(bytes20 liskAddress) public view returns (bool) {
+        return (airdropStatus[liskAddress] & STAKING_TIER_1_BIT) != 0;
+    }
+
+    /// @notice Check if the Lisk v4 address has claimed the airdrop for staking tier 2.
+    /// @param liskAddress The Lisk v4 address to check if it has claimed the airdrop for staking tier 2.
+    /// @return True if the Lisk v4 address has claimed the airdrop for staking tier 2, False otherwise.
+    function claimedStakingTier2(bytes20 liskAddress) public view returns (bool) {
+        return (airdropStatus[liskAddress] & STAKING_TIER_2_BIT) != 0;
+    }
+
+    /// @notice Check if the Lisk v4 address has claimed the full airdrop.
+    /// @param liskAddress The Lisk v4 address to check if it has claimed the full airdrop.
+    /// @return True if the Lisk v4 address has claimed the full airdrop, False otherwise.
+    function claimedFullAirdrop(bytes20 liskAddress) public view returns (bool) {
+        return (airdropStatus[liskAddress] & FULL_AIRDROP_CLAIMED) == FULL_AIRDROP_CLAIMED;
```

Comment:
Can we use `airdropStatus[liskAddress] == FULL_AIRDROP_CLAIMED` instead (I know this approach is referenced from L2Airdrop.sol, just curious)

Reply:
I would recommend not modifying any lines if not strictly necessary then what's inside `L2Airdrop.sol`, as this contract has been audited by a reputable firm and thoroughly battle-tested. It should be used as our baseline.

Reply:
It's the same with the original one.